adapters/audienceNetwork/facebook.go

```diff
@@ -447,3 +447,18 @@ func NewFacebookBidder(client *http.Client, platformID string, appSecret string)
 		appSecret:    appSecret,
 	}
 }
+
+func (fa *FacebookAdapter) MakeTimeoutNotification(req *adapters.RequestData) (*adapters.RequestData, []error) {
+	auction_id, err := jsonparser.GetString(req.Body, "imp", "[0]", "id")
+	if err != nil {
+		return &adapters.RequestData{}, []error{err}
+	}
+
+	timeoutReq := adapters.RequestData{
+		Method:  "GET",
+		Uri:     fmt.Sprintf("https://www.facebook.com/audiencenetwork/nurl/?partner=%s&app=%s&auction=%s&ortb_loss_code=2", fa.platformID, fa.platformID, auction_id),
```

Comment:
Consider moving the uri construction up a level. IMHO the extra indentation doesn't make the string substitutions any easier to see.